### DISCUSSION TITLE
inject html editor

### DISCUSSION THREAD

**Discussion by xi7** [gsantner/markor#2133] [Ideas] [answered]
add html inject editor to tools shortcut.
![Screenshot_20231017-024318_Markor_1](https://github.com/gsantner/markor/assets/67331785/7f928350-16e6-48b9-bd62-70115ec6be15)

**Comment by xi7** [comment]
![Screenshot_20231017-033400_Markor_1](https://github.com/gsantner/markor/assets/67331785/662e4ca1-a951-4733-813e-3fda9bd7c7a7)

also add html inject view aligned markdown columns.

**Reply by gsantner** [reply, marked as answer]
Not sure if I understand correct but seems like it's about the (width) sizing of your own content. 

For that you can add own your own local rules, specifically in your file or in the global CSS head injection.

**Reply by xi7** [reply]
the first idea is add quickly access injection because score change and we need close project and go to options.


about the css can u replicate the last pic and show me ?

**Reply by gsantner** [reply]
@xi7 
It is your content and up to you to customize it how you want it.  
<https://www.w3schools.com/css/css_table.asp>

**Reply by xi7** [reply]
working without injection using html.
